### PR TITLE
Fix for #1430 by first testing to see if Item2 of the Tuple is null

### PR DESCRIPTION
**Pull request by nickdevore**
Let me know if this is close to how you would have solved the problem.  It's my first attempt at contributing to this project.  I want to do things the way that fits with your coding style.

### CODE REVIEW

File: Sdk/Exceptions/AllException.cs [hunk 1]
```diff
@@ -48,7 +48,7 @@ public override string Message
                     return string.Format(CultureInfo.CurrentCulture,
                                          "{0}Item: {1}{2}{3}",
                                          indexString,
-                                         error.Item2.ToString().Replace(Environment.NewLine, spaces),
+                                         error.Item2 == null ? "null" : error.Item2.ToString().Replace(Environment.NewLine, spaces),
```
Comment (bradwilson) [inline Sdk/Exceptions/AllException.cs]: This should just be a null or empty string, not the word `null`.

**Review by bradwilson — CHANGES_REQUESTED** [review]

File: Sdk/Exceptions/AllException.cs [hunk 2]
```diff
@@ -48,7 +48,7 @@ public override string Message
                     return string.Format(CultureInfo.CurrentCulture,
                                          "{0}Item: {1}{2}{3}",
                                          indexString,
-                                         error.Item2 == null ? "null" : error.Item2.ToString().Replace(Environment.NewLine, spaces),
+                                         error.Item2?.ToString().Replace(Environment.NewLine, spaces),
```
Comment (bradwilson) [inline Sdk/Exceptions/AllException.cs]: I think you need `.Item2?.ToString()?.Replace` because calling .Replace on null will null-ref.

**Comment by nickdevore** [comment]
@bradwilson I made the change you asked for, though I did have a null in that spot in the unit test, and it didn't throw the null reference exception.  Perhaps [my test](https://github.com/xunit/xunit/pull/1582/commits/6627a5091aa1253f64c4f0dbb2ed35a0d942886f) doesn't cover the scenario you're talking about?

**Comment by nickdevore** [comment]
@bradwilson Just following up on this to make sure I did all that you wanted me to do on it.

**Comment by bradwilson** [comment]
Sorry, haven't been doing much xUnit.net work over the holidays. I will come back to this soon!

**Comment by bradwilson** [comment]
Thanks!

**Pull request merged**